src/main/resources/assets/carpet/scripts/ai_tracker.sc

```diff
@@ -381,6 +324,57 @@ global_functions = {
    },
 };
 
+global_hostile_to_villager = {
+   'drowned' -> 8,
+   'evoker'-> 12,
+   'husk' -> 8,
+   'illusioner' -> 12,
+   'pillager' -> 15,
+   'ravager' -> 12,
+   'vex' -> 8,
+   'vindicator' -> 10,
+   'zoglin' -> 10,
+   'zombie' -> 8,
+   'zombie_villager' -> 8
+};
+
+
+__config() ->{
+    'commands'->{
+        ''->'_command',
+        'clear'->'clear',
+        'toggle boxes'->_()->global_display_boxes = !global_display_boxes,
+        'toggle <display>'->['__toggle',null],
+        'toggle villager <villager_display>'->_(d)->__toggle('villager_'+d,null),
+        'toggle villager hostile_detection <hostile>'->_(h)->__toggle('villager_hostile_detection',h),
+        'update_frequency <ticks>'->_(ticks)->(global_interval = ticks;global_duration = ticks + 2),
+        'transparency <alpha>'->_(alpha)->global_opacity = floor(alpha)
```

Comment:
if alpha is in, doesn't need to be floored, right?

Reply:
ah ye ofc, just copied old code cos dumdum